using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -22,64 +25,73 @@ public static void syncIdentifier(final String identifierType,
-{% endhint %}
+The function of the `syncIdentifiers`  API is same as the `syncIdentifier` API, which passes a list of identifiers each containing an identifier type as the key and an identifier as the value. In each of the identifiers pair, both identifier type and identifier should be non empty and non null values, otherwise they will be ignore. 
 
 #### **Syntax**
 
 ```java
 public static void syncIdentifiers(final Map<String, String> identifiers,
-                                       final VisitorID.AuthenticationState authenticationState)
+                                   final VisitorID.AuthenticationState authState)
 ```
 
-#### **Example**
+o  *identifiers* map contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), that identifier is ignored by the Identity extension.
+
+o  *authState* indicates the authentication state for the user, which contains one of the VisitorID.AuthenticationState values `VisitorID.AuthenticationState.AUTHENTICATED`, `VisitorID.AuthenticationState.LOGGED_OUT`, `VisitorID.AuthenticationState.UNKNOWN`
+
+#### Example
 
 ```java
 Map<String, String> identifiers = new HashMap<String, String>();
-identifiers.put("idType", "idValue");
+identifiers.put("idType1", "idValue1");
+identifiers.put("idType2", "idValue2");
+identifiers.put("idType3", "idValue3");
 Identity.syncIdentifier(identifiers, VisitorID.AuthenticationState.AUTHENTICATED);
 ```
 
 ### syncIdentifiers \(overloaded\)
 
-**Tip**: All given customer IDs are given the default authentication state of `UNKNOWN`.
+This  API  `syncIdentifiers()` is an overloaded version that does not include the parameter for the Authentication State, and assumes a default value of `VisitorID.AuthenticationState.UNKNOWN`
 
-These IDs are preserved between app upgrades, are saved and restored during the standard application backup process, and are removed at uninstall. If the current SDK privacy status is `optedout`, calling this method results in no operations being performed.
-
-{% hint style="info" %}
-The `identifiers` dictionary contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
-{% endhint %}
+These IDs are preserved between app upgrades, are saved and restored during the standard application backup process, and are removed at uninstall. 
 
 #### **Syntax**
 
 ```java
 public static void syncIdentifiers(final Map<String, String> identifiers);
 ```
 
-#### **Example**
+o  *identifiers* is a map that contains the Identifiers with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
```

Comment:
- Please remove the "o" and just keep the asterisk.

- Please start the 2nd sentence on a new line and line it up with *identifiers* from the previous line. 
Please reword the 2nd sentence so that it reads as follows:

"If the identifier type and identifier have null or empty values, the identifier is ignored by the Identity extension."